scripts/xnat/check_valid_sessions.py

```diff
@@ -13,19 +13,43 @@
 
 Example
 =======
-- Update the cache and generate the baseline report
-  ./check_valid_sessions --update --baseline
+- When running for the first time run 
+  ./check_valid_sessions.py --update 
+  so that the cach (located at experimentsdir) is created 
+
+- Update the cache (stored in experimentsdir) and generate the baseline report
+  ./check_valid_sessions.py --update --baseline
 
 - Use the existing cache to extract 10 in the followup window
- ./check_valid_sessions --num_extract 10 --min 180 --max 540
+ ./check_valid_sessions.py --num_extract 10 --min 180 --max 540
+
+If you have problems running it log in as nicholsn and run 
+module load miniconda
+source activate ncanda-1.0.0
 """
 __author__ = "Nolan Nichols <http://orcid.org/0000-0003-1099-3328>"
 __modified__ = "2015-08-26"
 
 import os
 
+import re
+
+def cmpVersion(version1, version2):
+    def normalize(v):
+        return [int(x) for x in re.sub(r'(\.0+)*$','', v).split(".")]
+    return cmp(normalize(version1), normalize(version2))
```

Comment:
please remove this function from the script, as we discussed versions should be specified in a top level requirements.txt field